.travis.yml

```diff
@@ -1,33 +1,61 @@
 language: php
 
-sudo: false
-
 cache:
   directories:
     - $HOME/.composer/cache/files
 
+branches:
+    only:
+        - master
+
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+  - '5.5'
+  - '5.6'
+  - '7.0'
+  - '7.1'
+  - nightly
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env: DEPS=dev
```

Comment:
we should keep the deps option. When testing, a couple things are important to test:

 * Lowest version within the constraint (5.0)
 * All minor versions within the constraint (5.2)
 * The dev versions (5.2-dev). When installing this bundle in a project with `minimum-stability: dev`, the dev dependencies will be installed. We should still verify that the bundle is working in this condition.